docs/screens/authenticating.md

```diff
@@ -0,0 +1,27 @@
+# Authenticating
+
+## Documentation
+
+| When authentication started. | <pre>{<br />   "name": "action_start"<br />   "item_name": "authentication"<br />}<pre/> |
+|:----------------------------:|------------------------------------------------------------------------------------------|
+
+| When authentication finished with success. | <pre>{<br />   "name": "action_finished"<br />   "item_name": "authentication"<br />   "success": true<br />}<pre/> |
+|:------------------------------------------:|---------------------------------------------------------------------------------------------------------------------|
+
+| When authentication finished with failure. | <pre>{<br />   "name": "action_finished"<br />   "item_name": "authentication"<br />   "success": false<br />}<pre/> |
+|:------------------------------------------:|----------------------------------------------------------------------------------------------------------------------|
+
+| When get access token started. | <pre>{<br />   "name": "action_start"<br />   "item_name": "get_access_token"<br />}<pre/> |
+|:------------------------------:|--------------------------------------------------------------------------------------------|
+
+| When get access token finished with success. | <pre>{<br />   "name": "action_finished"<br />   "item_name": "get_access_token"<br />   "success": true<br />}<pre/> |
+|:--------------------------------------------:|-----------------------------------------------------------------------------------------------------------------------|
+
+| When get access token finished with failure. | <pre>{<br />   "name": "action_finished"<br />   "item_name": "get_access_token"<br />   "success": false<br />}<pre/> |
+|:--------------------------------------------:|------------------------------------------------------------------------------------------------------------------------|
+
+| When login error is shown. | <pre>{<br />   "name": "screen_opened"<br />   "item_name": "login_error"<br />}<pre/> |
```

Comment:
**chore**: "screen_opened" should be called, when we open given screen, No when we get result of loading data. The result of loading data is tracked by "action_start" and "action_finished". 

I think it would be fine to just have:
```json
{
   "name": "screen_opened"
   "item_name": "authenticaticating_screen"
}

```

Reply:
This is not fixed